guides/serialization_and_saving.py

```diff
@@ -1,92 +1,64 @@
 """
-Title: Serialization and saving
+Title: Save and Load Models
 Authors: Kathy Wu, Francois Chollet
 Date created: 2020/04/28
 Last modified: 2020/04/28
-Description: Complete guide to saving & serializing models.
+Description: Complete guide to saving & loading models.
 """
 
 """
 ## Introduction
 
 A Keras model consists of multiple components:
 
-- An architecture, or configuration, which specifyies what layers the model
-contain, and how they're connected.
-- A set of weights values (the "state of the model").
-- An optimizer (defined by compiling the model).
-- A set of losses and metrics (defined by compiling the model or calling
-`add_loss()` or `add_metric()`).
+- An architecture, that specifies the layers and how they're connected.
+- A set of trained parameters, such as weights and biases.
+- The compilation information, that includes an optimizer, losses and metrics.
 
-The Keras API makes it possible to save all of these pieces to disk at once,
-or to only selectively save some of them:
+The Keras API makes it possible to save and load all or selectively some of these parts:
 
-- Saving everything into a single archive in the TensorFlow SavedModel format
-(or in the older Keras H5 format). This is the standard practice.
-- Saving the architecture / configuration only, typically as a JSON file.
-- Saving the weights values only. This is generally used when training the model.
+- Save and Load model (**recommended**), i.e, everything in the SavedModel format (or the older H5 format).
+- Save and Load architecture only, typically as a JSON file.
+- Save and Load trained parameters only, typically as checkpoints.
 
-Let's take a look at each of these options: when would you use one or the other?
-How do they work?
+Let's take a look at each of these options in detail:
 """
 
 """
-## The short answer to saving & loading
+## Save and Load Model
 
-If you only have 10 seconds to read this guide, here's what you need to know.
+#### If you only have 10 seconds to read this guide, here's what you need to know.
```

Comment:
Google style says "don't skip heading levels" and "don't leave empty heading-sections".

So maybe this H4 should just be a bold for emphasis.

Reply:
Updated this to bold.